environments/prod/prod.tfvars

```diff
@@ -553,11 +553,47 @@ frontends = [
       }
     ]
   },
+  {
+    name             = "pip-frontend-b2c-sign-in"
+    custom_domain    = "sign-in.court-tribunal-hearings.service.gov.uk"
+    backend_domain   = ["hmctspipprod.b2clogin.com"]
```

Comment:
To fix host header:
```suggestion
    backend_domain   = ["hmctspipprod.b2clogin.com"]
    host_header           = "hmctspipprod.b2clogin.com"
```

Reply:
Not sure what's going on with this suggestion it lines up properly in my comment.